api/accounting/encoding.go

```diff
@@ -0,0 +1,82 @@
+package accounting
+
+import (
+	"github.com/nspcc-dev/neofs-sdk-go/internal/proto"
+)
+
+const (
+	_ = iota
+	fieldDecimalValue
+	fieldDecimalPrecision
+)
+
+// MarshaledSize returns size of the Decimal in Protocol Buffers V3 format in
```

Comment:
is it handmade? there was some concept with code generation, have you considered it? tree things were made with it: https://github.com/nspcc-dev/neofs-node/blob/22259ce755199d6e87331a3b58cc14449a36f69e/pkg/services/tree/types_neofs.pb.go

Reply:
> have you considered it?

not yet, we need a proper testing 1st (it's coming with this PR). Then we can experiment with codegen

Reply:
imo, a generator should have been written first, that is an idea about it. but it is late